eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -2,7 +2,7 @@ parameters:
   SDKType: client
   ServiceDirectory: not-specified # Set a default that breaks in obvious ways.
   PreTestSteps: []
-  TestOptions: '$(DefaultOptions) -Dmaven.javadoc.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true'
+  TestOptions: '$(DefaultOptions) -Dmaven.javadoc.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true'
```

Comment:
Should jacoco be skipped here as well?

Reply:
We need the jacoco report generated here because we want to fail the build if coverage drops below the threshold.